spec/suites/layer/PopupSpec.js

```diff
@@ -229,6 +229,18 @@ describe('Popup', function () {
 		L.Icon.Default.prototype.options.popupAnchor = popupAnchorBefore;
 	});
 
+	it("prevents an underlying map click for Layer", function () {
```

Comment:
Those tests are testing what the code is actually doing, while I think it would be better to test what a user should expect. So instead I would make sure the path popup is open, no matter how.
Something like `map.hasLayer(path._popup)`.
What do you think?